### DISCUSSION TITLE
Overview of source tree and library / plugin components

### DISCUSSION THREAD

**Discussion by rlei-weta** [Autodesk/maya-usd#875] [Q&A] [answered]
Hello!

Is there a document available with an overview of maya-usd's source tree, what the various library / plugin components are responsible for, and especially what user facing features they contribute to?

Happy to read through the code - but an overview would also be useful :)

**Comment by kxl-adsk** [comment, marked as answer]
I'm afraid we don't have anything yet available. Please don't hesitate to ask questions...and maybe this will be the right push to start adding README.md files to specific folders in the source tree.

**Reply by rlei-weta** [reply]
Thanks @kxl-adsk!  I realize there's still moving pieces, and docs may spell extra maintenance. 

Will definitely have questions :smile:
